msg/km_KH.utf8/cubrid.msg

```diff
@@ -1329,7 +1329,7 @@ Check the path of the key file (_keys) and make sure that it includes proper key
 1258 Cannot find the key set on the database from %1$s. The key file from backup volume is going to be used.
 1259 Rename the key file from "%1$s" to "%2$s".
 1260 Copy the key file "%1$s" to "%2$s".
-1261 Cannot find the key at the time to restore (key index: %1$d, created time: %2$s). Set the key (key index: %3$d) for the database.
+1261 The first key (key index: %1$d, created time: %2$s) on the restored backup key file has been set.
```

Comment:
I think 'restored key file from the backup key file' is the correct expression. right? But, it is too long. So, how about 'restored key file' instead of 'restored backup key file'?

Reply:
OK!